models/contacts.js

```diff
@@ -1,19 +1,104 @@
-// const fs = require('fs/promises')
+const { readJsonFile, writeJsonFile } = require("../utils/fileOperations");
 
-const listContacts = async () => {}
 
-const getContactById = async (contactId) => {}
+const { safeJoin } = require("../utils/path");
 
-const removeContact = async (contactId) => {}
+const contactsPath = safeJoin(__dirname, "contacts.json");
 
-const addContact = async (body) => {}
+const listContacts = async () => {
+  try {
+    const list = await readJsonFile(contactsPath);
+    return list;
+  } catch (error) {
+    console.log(error);
+  }
+};
 
-const updateContact = async (contactId, body) => {}
+const getContactById = async (contactId) => {
+  try {
+    const list = await listContacts();
+    const contact = list.find((c) => Number(c.id) === Number(contactId));
+
+    return contact;
```

Comment:
- zmienna `list` to raczej `contacts` (dokladniej).
- mozesz zwrocic `.find` bezposrednio (nie ma potrzeby robic `contact`).
- dwa razy robisz `Number`, wiec zgaduje, ze kazde `id` to String. Po co kazde z nich przeksztalcac w inne dane? o_O